float64 speed_is
float64[4] steering_is
float64[3] distance_frot
float64[3] distance_back
int8[3] error_id
string  error_message
